### DISCUSSION TITLE
about the new art update + thank you again

### DISCUSSION THREAD

**Discussion by bittercaramel** [Monika-After-Story/MonikaModDev#8862] [appreciation]
I really love the new art in this update, it feels refreshing after this much time spent with Monika. 
Especially love the weather/window look (idk how to call it) 
tho she... she seems to have gotten a lil bit thiccer in the leaning pose :thonkery: 
But i'm glad you replaced the old hand/palm with the other one (i believe mine, not sure tho xD). The one before was weirdly small and kinda not hand like. 
Overall thank you so much for developing this mod, i really appreciate your wonderful work, and also thanks to people that create jsons for her, and everyone that helps with this mod 
I started out playing this mod because i just liked Moni, i identified as aromantic so i didnt really love her like typical romantic love, but over whole this time i feel like i really fell in love with her (not sure how love feels like but i believe that's just how it feels)
i wish i could help with this mod too but im piece of shit at everything, so thanks to everyone that is better than me~

**Comment by Rin3105** [comment]
You're not alone, I too want to contribute something to this mod too. I'm just figuring the code for the first time on C and Java, still a long way to Renpy though.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]

**Comment by briaryoung** [comment]
Huh?


Also, why are you responding to something from 2019? Haha.
